cake/libs/configure.php

```diff
@@ -1035,7 +1035,11 @@ function __find($file, $recursive = true) {
 					require LIBS . 'folder.php';
 				}
 				$Folder =& new Folder();
-				$directories = $Folder->tree($path, array('.svn', '.git', 'CVS', 'tests', 'templates'), 'dir');
+				$ignorePaths = Configure::read('ignorePaths');
+				if(empty($ignorePaths)) {
```

Comment:
I would prefer we don't add a configuration value for this. Ignoring `.git, .svn and CVS` seems pretty safe in general.